cards/src/main/resources/cards/custom/group3/set_1087/minion_irena_dragon_knight.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "Dragon Caretaker",
+  "baseManaCost": 3,
+  "type": "MINION",
+  "heroClass": "RUST",
+  "baseAttack": 2,
+  "baseHp": 1,
+  "rarity": "LEGENDARY",
+  "description": "Blitz. Adjacent Dragons have double their Attack.",
+  "aura":
+    {
+      "class": "Aura",
```

Comment:
This should probably be a `StatefulAttributeValueAura`. Use the `AURA_ATTACK_MULTIPLIER` as the attribute, and remove the apply/remove effect and remove always apply.